packages/polars/meta.yaml

```diff
@@ -0,0 +1,18 @@
+package:
+  name: polars
+  version: 1.18.0
+  top-level:
+    - polars
+# TODO: Upgrade to Rust Nightly 2024-12-19 when adding the sdist instead of the wheel
+source:
+  url: https://github.com/pola-rs/polars/releases/download/py-1.18.0/polars-1.18.0-cp39-abi3-emscripten_3_1_58_wasm32.whl
+  sha256: bbf7dd42bedf2f6889aba86220f43b9f09c3e282dc8fbf602bc14519152b2440
+about:
+  home: https://pola.rs/
+  PyPI: https://pypi.org/project/polars
+  summary: Blazingly fast DataFrame library
+  license: MIT
+extra:
+  recipe-maintainers:
+    - georgestagg
+    - ritchie46
```

Comment:
@georgestagg and @ritchie46, I hope you'd be okay with being added here? Adding maintainers for recipes helps us with problems that can arise with updating the package version and helps users know who to contact in case they may face any bugs with Polars running in Pyodide/WASM.


xref: #4506

Reply:
Yes!

Reply:
Sure :)